src/main/resources/templates/db_scripts/schema.json

```diff
@@ -70,12 +70,12 @@
     {
       "run": "after",
       "snippetPath": "default_data/create_default_mapping_profile.sql",
-      "fromModuleVersion": "mod-data-export-2.1.0"
+      "fromModuleVersion": "mod-data-export-2.2.0"
```

Comment:
let this be 2.1.4, our next release version

Reply:
done)